app/src/main/java/com/tpp/theperiodpurse/AppUiState.kt

```diff
@@ -7,5 +7,8 @@ import com.tpp.theperiodpurse.data.Symptom
 data class AppUiState (
     // Preffered Symptoms to be tracked
     var trackedSymptoms: List<Symptom> = listOf(),
+    val allowReminders: Boolean = false,
+    val reminderFrequency: String = "Every day",
+    val reminderTime: String = "12:00 PM",
```

Comment:
Looks like none of these ui states are reflected in the actual view. i.e. when I set to be notified at 2PM and restart the app, the view still says I will be notified at 12PM. Are you loading these from the database inside `appViewModel` ? Look at `loadData` in `AppViewModel`